package.json

```diff
@@ -39,17 +39,18 @@
     "jison": "0.4.17",
     "logops": "1.0.0",
     "moment": "^2.19.4",
+    "moment-timezone": "^0.5.21",
```

Comment:
In order to avoid problems with "open" version dependencies, we prefer to use ~ instead of ^

(Some of pre-existing dependencies are yet using ^. We changed this in other repos but it seems this one was forgotten. Pre-existing dependencies will be fixed in a next PR).

Reply:
Fixed with 0f341d164437b4f2b9876ba8acf56d25afa5cefa